jsonrpcbase/main.py

```diff
@@ -448,8 +425,8 @@ def _validate_params_types(self, method, params):
 
             for key in params.keys():
                 if key not in self.method_data[method]['types'] or \
-                        not (isinstance(params[key], self.method_data[method]['types'][key]) or
-                             params[key] is None):
+                        not (isinstance(params[key], self.method_data[method]['types'][key])
+                             or params[key] is None):  # noqa
```

Comment:
what do you think about being more specific here? e.g. `# noqa: W503` (or W504)?

Reply:
ah, I see now that a [later commit](https://github.com/kbaseIncubator/jsonrpcbase/pull/1/commits/f95885d8c593a26f83caec360cc05791a96e1b58) removes the `# noqa` completely;